doc/man3/SSL_set_async_callback.pod

```diff
@@ -0,0 +1,81 @@
+=pod
+
+=head1 NAME
+
+SSL_set_async_callback,
+SSL_get_async_status,
+SSL_clear_async_status
+- manage asynchronous operations
+
+=head1 SYNOPSIS
+
+=for comment multiple includes
+
+ #include <openssl/ssl.h>
+
+ int SSL_set_async_callback(SSL *s, ASYNC_callback_fn callback,
+                            void *callback_arg);
+ int SSL_get_async_status(SSL *s, int *status);
+ int SSL_clear_async_status(SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_set_async_callback() allows SSL application to set a callback in
+asynchronous SSL session, so that when engine completed a cryptography
+operation, the callback can be called to notify application to resume the
+paused work flow.
+
+SSL_get_async_status() returns the engine status. This function facilitates
+the communication from engine to application. During SSL session, cryptographic
+operation would be dispatched to engine, and engine status is very useful for
+application to know if engine has successfully dispatched it to hardware.
+
+SSL_clear_async_status() help SSL application to clear the EAGAIN status if
```

Comment:
EAGAIN means "error again".  It is generated when the engine to tell user code to try again. 
But I think it is better to changing this word to "engine", so that it can cover all situations.